src/main/java/mops/presentation/GroupController.java

```diff
@@ -56,7 +52,7 @@ public String getAllFilesOfDirectory(KeycloakAuthenticationToken token,
     public String searchFilesInGroup(KeycloakAuthenticationToken token,
                                      Model model,
                                      @PathVariable("groupId") int groupId,
-                                     @Param("searchQuery") FileQuery query) {
+                                     @RequestAttribute("searchQuery") FileQuery query) {
```

Comment:
I think we should use `@ModelAttribute` here.
https://stackoverflow.com/questions/29370581/spring-mvc-please-explain-difference-between-requestparam-and-modelattribute
https://www.baeldung.com/spring-mvc-and-the-modelattribute-annotation

Reply:
I don't think it worked. This is `@ModelAttribute`
![image](https://user-images.githubusercontent.com/25705862/76152888-28598280-60c5-11ea-895d-a34567c35bcc.png)

and this `@RequestAttribute`
![image](https://user-images.githubusercontent.com/25705862/76152892-3909f880-60c5-11ea-94b9-118da0380082.png)

Reply:
Yes when using @ModelAttribute it will try to create the FileQuery object with its fields from the request parameters.
So if FileQuery got the field "name" it will look for the name request parameter

Reply:
So it is working but not in this test context? I mean as the tests does not fail we could use `@ModelAttribute` if it makes life easier later on.

Reply:
I missed one `@RequestAttribute`. It acutally does not work. 

![image](https://user-images.githubusercontent.com/25705862/76153016-8470d680-60c6-11ea-863a-e5ba44eb3bc6.png)

Reply:
Yeah it wont work for primitive objects, as there no request parameters for the fields of String: `byte[] value`, `byte code`, `int hash` & `boolean hashIsZero`

Reply:
Ok thanks for clearing that up.